blog/_posts/2017-09-08-team_code_hoppers_blog.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Our journey - with rgsoc and Open Source!
+layout: post
+created_at: Thursday Sep 08 2017
```

Comment:
Oops! You added two extra files [one is a copy of the blog post fiel and one is Cythia's picture]. Can you please delete them? Also, the day and date doesn't match. I would have changed it by myself but as you'll anyway need to send the commit for other changes it would be good if you can fix this as well. As now we are planning to publish this today, you would need to change it to Saturday Sep 09 2017. Please make sure to change it at all 3 places. :)